pkg/operator/mgr/mgr.go

```diff
@@ -187,6 +239,14 @@ func (c *Cluster) createKeyring(clusterName, name string) error {
 	return nil
 }
 
+// Ceph docs about the prometheus module: http://docs.ceph.com/docs/master/mgr/prometheus/
+func (c *Cluster) enablePrometheusModule(clusterName string) error {
```

Comment:
the metrics documentation/example should be updated to point at this new ceph-mgr module endpoint instead.

https://github.com/rook/rook/blob/master/Documentation/k8s-monitoring.md
https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/monitoring/service-monitor.yaml#L14

Reply:
The docs don't mention the API nor the Ceph mgrs right now, should that be added?